src/CoursePageResources/CoursePageResources.test.jsx

```diff
@@ -1,4 +1,4 @@
-describe('example', () => {
+describe('coursepageresources', () => {
   it('will pass because it is an example', () => {
 
   });
```

Comment:
Add some snapshot tests.

Reply:
Note - we'd like to avoid overuse of snapshot tests, as while they're easy to write, they're brittle and difficult to maintain.  Favoring specific assertions about the DOM is preferable, as it's much less likely to break when trivial changes are made to the markup/CSS classnames.  

Snapshots are good for small components, but not great for larger components.  i.e., the cost of ensuring that the snapshot is always 100% correct is so high for large components that it's very easy for the snapshot to become invalid, in which case the test starts asserting that things are still wrong, but no one realizes because it passes. :)

A large, invalid snapshot is very difficult to go back and prove wrong because it requires a developer to go understand every little thing that's in the snapshot, and to go against what the machine is telling them - that the test passed.  Visual inconsistencies are better caught with our eyes when viewing the UI, rather than trying to parse a not-very-human-readable HTML document.  

Some good rationale in here: https://engineering.ezcater.com/the-case-against-react-snapshot-testing

Reply:
Thanks for this explanation! Definitely useful to keep in mind for future PRs as well.

Reply:
What are snapshot tests? Or maybe better, can you point me to some examples? Will I find some in frontend-app-learning?

Reply:
Just read David Joy's comment (my page was out of date) and I'm inclined to agree that meaning is more important than appearance (to paraphrase).

Reply:
Snapshot tests record a copy of the DOM generated from react when they're first run, and then on subsequent runs compare the results to that original snapshot.  It's like asserting that the entire DOM is what you expect it to be, down to every single tag and attribute.  As mentioned above, this means that any tiny change no matter how trivial invalidates a snapshot test and can make noisy diffs - often this can result in masking _real_ issues if the test is "retrained" under faulty assumptions about what's correct.

You can see some in this commit in frontend-app-learning: https://github.com/edx/frontend-app-learning/pull/97/commits/ada07504fdd6bf5ab4a4f2f68491f7f53a160416

Note that I've asked that those be modified (in the same spirit as above), so that code may not make it into master. :)

Reply:
Yeah, I read the rational and couldn't agree more. I had done similar by hand years ago doing compiler testing and hated it, thus avoided doing automated tests until I figured out how to do meaningful tests. More work to write the tests, but they are very much more reliable.